p2p/message.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package p2p
+
+import (
+	"encoding/binary"
+	"io"
+
+	common "github.com/ChainSafe/gossamer/common"
+)
+
+const (
```

Comment:
Can we please use go const naming conventions :heart:

Reply:
```
const (
	StatusMsg        = iota
	BlockRequestMsg
	BlockResponseMsg
	BlockAnnounceMsg
	TransactionMsg
	ConsensusMsg
	RemoteCallRequest
	RemoteCallResponse
	RemoteReadRequest
	RemoteReadResponse
	RemoteHeaderRequest
	RemoteHeaderResponse
	RemoteChangesRequest
	RemoteChangesResponse
	ChainSpecificMsg          = 255
)
```

Reply:
done